src/openms/include/OpenMS/ANALYSIS/OPENSWATH/MRMFeatureFilter.h

```diff
@@ -94,13 +95,81 @@ namespace OpenMS
       const TargetedExperiment & transitions);
 
     /**
-      @brief Converts a FeatureMap to a qcMLFile::Attachment
+      @brief Flags or filters features and subordinates in a FeatureMap based on a user
+        defined set of filter values derived from calling `EstimatePercRSD`.  The user supplied
+        filter_criteria represents the bounds on acceptable %RSD values.
+        NOTE that flagging nor filtering will be done on the labels and transitions type counts.
 
       @param features FeatureMap to flag or filter
-      @param attachment qcML Attachment
+      @param filter_criteria MRMFeatureQC class defining QC parameters defining the acceptable limits of the %RSD
+        where %RSD = (value std dev)/(value mean)*100%
+      @param filter_values MRMFeatureQC class filled with bounds representing the %RSD found in e.g., pooled QC samples or replicate Unknown samples
 
     */
-    void FeatureMapToAttachment(FeatureMap& features, QcMLFile::Attachment& attachment);
+    void FilterFeatureMapPercRSD(FeatureMap& features, const MRMFeatureQC& filter_criteria, const MRMFeatureQC& filter_values);
+
+    /**
+      @brief Flags or filters features and subordinates in a FeatureMap based on a user
+        defined set of filter values derived from calling `EstimateBackgroundInterferences`.  The user supplied
+        filter_criteria represents the bounds on acceptable %BackgroundInterference values.
+        NOTE that filtering is only done on the `Intensity` member.
+
+
+      @param features FeatureMap to flag or filter
+      @param filter_criteria MRMFeatureQC class defining QC parameters defining the acceptable limits of %BackgroundInterference
+        where %BackgroundInterference = (value Sample)/(value Blank)*100%
```

Comment:
could you check if this actually generates the desired documentation? `%` and doxygen gives me the creeps

Reply:
I changed it just in case.